packages/mgt/src/graph/graph.people.ts

```diff
@@ -195,28 +195,44 @@ export async function getPeople(graph: IGraph): Promise<Person[]> {
  * async promise to the Graph for People, defined by a group id
  *
  * @param {string} groupId
+ * @param {boolean} [transitive=false] - whether the return should contain a flat list of all nested members
+ * @param {PersonType} [personType=PersonType.person] - the type of person to search for
  * @returns {(Promise<Person[]>)}
  * @memberof Graph
  */
-export async function getPeopleFromGroup(graph: IGraph, groupId: string): Promise<Person[]> {
+export async function getPeopleFromGroup(
```

Comment:
You know, `getPeopleFromGroup` and `findUsersFromGroup` are very similar, should we just combine them? `findUsersFromGroup` could act as `getPeopleFromGroup` when no query is provided.

Reply:
I think it would be a good idea... It can be somewhat confusing to talk users vs. person. vs. people 😊 I'll do the changes early next week probably!